shared/reducers/users.js

```diff
@@ -32,7 +32,8 @@ const reducer = (
       return state.update('infoMap', map =>
         map.withMutations(m => {
           newlyFixed.forEach(user => {
-            m.update(user, info => (info || blankUserInfo).set('broken', false))
+            // only make it if one exists already
```

Comment:
because we only care if the user is broken?

Reply:
else we get these messages about 'fixed' people who we never thought were broken, so we mutate the store a ton while this is essentially a broken only list (if you're not in here we don't consider you broken)